src/Mouf/Packanalyst/Services/FetchDataService.php

```diff
@@ -5,6 +5,7 @@
 use Composer\Downloader\DownloadManager;
 use Mouf\Packanalyst\ClassesDetector;
 use Composer\Package\Package;
+use Composer\Package\PackageInterface;
```

Comment:
this is needed for the ``/* @var $package PackageInterface */`` which is in the code